docs/seqcols/compare_collections.md

```diff
@@ -54,23 +60,7 @@ How to assess: We will ignore the `sequences` attribute, but ensure that the `na
 
-### Others...
-
-There are also probably other types of compatibility you can assess using the result of the `/comparison` function.
-Now that you know the basics, and once you have an understanding of what the comparison function results mean, it should be possible to figure out if you can assess a particular type of compatibility for your use case.
-
-## Complex cases: distinguishing out-of-order from mismatched arrays
-
-One challenge is to identify issues where the *set* of sequences and names both match, but some of the pairs have been swapped.
-For example, what would the comparison function return for two sequence collections that have the same content, but in different orders, AND where in addition two of the sequences have swapped names?
-
-Because the sequence array would contain the same sequences, the comparison function will count them all as matching.
-Similarly, the names arrays contains the same names, and so all will be counted as a match.
-However, the same_order will *not* be true; it will yield false for some of the attributes.
-
-This is the same output as a comparison of two sequence collections in different orders, without the name swap. This is a fundamental limitation of the array-based method of comparing.
-
-In this particular example, these results can be distinguished by the `sorted_name_length_pairs` attribute, because this would yield a perfect match for the second example, where all the pairs are intact but in a different order -- but it would NOT yield a match for the example with swapped names, because the name-length pairs would be different.
+## Complex cases
 
-This solves the issue for swapped names, but there is still potential for problems with other arrays or custom attributes. Therefore, we warn users that when the `_same_order` is flagged as false, this *does not imply that the pairs are intact*, and if this is a requirement, further investigation would be necessary. If distinguishing these scenarios is important, one possible solution would be to add another non-inherent collated attribute, similar to `sorted_name_length_pairs`, but including *all* collated attributes for each element rather than just the names and lengths. The comparison function would then immediately provide an answer as to whether the annotated sequence elements match *as units* between two collections.
+For more complex cases, the comparison function and the level1 digests can sometimes be used to figure out what is going on, but they are limited by design -- for situations that are more complex than these methods can handle, it is always possible to look deeper at the contents of the sequence collection and compare them directly. 
```

Comment:
Looks good. I have one suggestion, however: Could we still add a note at the end with the saying something like:

"Note that the `/comparison` endpoint only tests the order of each array attribute independently. There is currently no general test of order consistency across several array attributes, e.g. whether a single set of collated values for `names`, `lengths`, and `sequences` retains the same index across all three arrays if reordered. A concrete example for interpreting such a case will be added later."

I think it is important that this is at least mentioned somewhere.